services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -307,225 +394,212 @@ public Object answer() throws Throwable {
 
-        individualRequest.setRequestStatus("Forwarded");
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall().times(1);
+        individualRequest.setRequestStatus(FORWARDED);
 
         final IGlobalParameter connectionTimeOutParam = new GlobalParameter();
-        connectionTimeOutParam.setName("TARGET_APP_TIMEOUT");
+        connectionTimeOutParam.setName(TARGET_APP_TIMEOUT);
         connectionTimeOutParam.setValue("1000");
-        EasyMock.expect(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_TIMEOUT")).andReturn(
+        when(this.mockCacheable.getValue(IGlobalParameter.class, TARGET_APP_TIMEOUT)).thenReturn(
                 connectionTimeOutParam);
 
         final IGlobalParameter receiveTimeOutParam = new GlobalParameter();
-        receiveTimeOutParam.setName("TARGET_APP_RESP_TIMEOUT");
-        receiveTimeOutParam.setValue("12000");
-        EasyMock.expect(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_RESP_TIMEOUT")).andReturn(
+        receiveTimeOutParam.setName(TARGET_APP_RESP_TIMEOUT);
+        receiveTimeOutParam.setValue(TWELVE_THOUSAND);
+        when(this.mockCacheable.getValue(IGlobalParameter.class, TARGET_APP_RESP_TIMEOUT)).thenReturn(
                 receiveTimeOutParam);
 
         final SoapFaultException soapEx = new SoapFaultException("Soap Fault", "Soap Fault occurred");
         this.mockConsumerGateway.individualRequest(individualRequest, 1000, 12000);
-        EasyMock.expectLastCall().andThrow(soapEx);
+        doThrow(soapEx).when(mockConsumerGateway).individualRequest(individualRequest, 1000, 12000);
 
         this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
+        verify(mockIndividualRequestDao).persist(individualRequest);
 
-        this.mockMessageWriter.queueMessage(EasyMock.isA(ISdtMessage.class), EasyMock.isA(String.class),
-                EasyMock.eq(true));
-        EasyMock.expectLastCall();
+        this.mockMessageWriter.queueMessage(isA(ISdtMessage.class), isA(String.class), eq(true));
+       // verify(mockMessageWriter).queueMessage(isA(ISdtMessage.class), isA(String.class), eq(true));
 
-        EasyMock.replay(mockIndividualRequestDao);
-        EasyMock.replay(mockConsumerGateway);
-        EasyMock.replay(mockMessageWriter);
-        EasyMock.replay(mockCacheable);
+        ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
+        listAppender.start();
+        logger.addAppender(listAppender);
 
         this.targetAppSubmissionService.processRequestToSubmit(sdtRequestRef);
 
-        EasyMock.verify(mockIndividualRequestDao);
-        EasyMock.verify(mockConsumerGateway);
-        EasyMock.verify(mockMessageWriter);
-        EasyMock.verify(mockCacheable);
+        List<ILoggingEvent> logList = listAppender.list;
 
-        Assert.assertEquals("Individual Request status not as expected",
+        assertFalse(logList.isEmpty());
```

Comment:
If the test expects specific messages to be written to the log it should check for them rather than just checking it contains any messages.

Reply:
Test is still just checking if any messages have been written to the log rather than any specific messages.  Might be worth checking the test to see if there's any need to check the log in this test - if not then it might be worth removing the log checks.

Reply:
done